lib/zomato.rb

```diff
@@ -0,0 +1,98 @@
+require 'net/http'
+
+$key = ENV['ZOMATO_API_KEY']
+$base_url = 'https://developers.zomato.com/api/v2.1'
+
+class Zomato
+
+  def initialize(city_id)
+    @city_id = city_id
+  end
+
+  def createCuisines
```

Comment:
You can make code much simpler if you'll separate this logic.
"Pure" Zomato ApiClient in `lib/zomato` (only requesting specific endpoints, handle errors and return responses). 
And separate "[Service](https://medium.freecodecamp.org/service-objects-explained-simply-for-ruby-on-rails-5-a8cc42a5441f)" class

Reply:
@setevik where would you place this service? In the same lib folder?

Reply:
`app/services`.

Code in `/lib` usually not aware of application models / business logic